Header header
int32 stateSize
int32 id
float64[] P_hat
float64[] X_hat
float64[] s
float64[] y

